voice/record-a-call.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+from flask import Flask, request, jsonify
+from pprint import pprint
+
+app = Flask(__name__)
+
+@app.route("/webhooks/answer")
+def answer_call():
+    ncco = [
+        {
+            "action": "talk",
+            "text": "Hi, we will shortly forward your call. This call is recorded for quality assurance purposes."
+        },
+        {
+            "action": "record",
+            "eventUrl": ["https://demo.ngrok.io/webhooks/recordings"]
+        },
+        {
+            "action": "connect",
+            "eventUrl": ["https://demo.ngrok.io/webhooks/event"],
+            "from": "NEXMO_NUMBER",
+            "endpoint": [
+                {
+                    "type": "phone",
+                    "number": "RECIPIENT_NUMBER"
+                }
+            ]
+        }
+    ]
+    return jsonify(ncco)
+
+@app.route("/webhooks/recordings", methods=['POST'])
+def recordings():
+    data = request.get_json()
+    pprint(data)
+    return ("200")
```

Comment:
I feel this is confusing, for a few reasons:
* The parentheses aren't needed, and imply a tuple (which it isn't because it would need a trailing comma)
* The string `"200"` will be converted to the _body_ of the response, but it _looks_ like the HTTP status code.
* The HTTP 200 status code is implicit :)

I'm not sure what content would actually be useful, given that it's ignored by Nexmo's servers, but something more informative like the following may be more clear:
```python
return "Webhook received"
```